plugins/@grouparoo/salesforce/__tests__/export-objects/destination-options.ts

```diff
@@ -310,4 +316,68 @@ describe("salesforce/sales-cloud/destinationOptions", () => {
     expectFullResult(result);
     expectLeadTopicResult(result, 2);
   });
+
+  test("cannot sync if only one Reference option is set", async () => {
+    const badDestinationOptions = {
+      recordObject: "Contact",
+      recordMatchField: "Custom_External_ID__c",
+      recordReferenceField: "AccountId",
+    };
+
+    await expect(
+      exportBatch({
+        appId,
+        appOptions,
+        destinationOptions: badDestinationOptions,
+        syncOperations,
+        exports: [
+          {
+            recordId: id1,
+            oldRecordProperties: {},
+            newRecordProperties: {
+              Custom_External_ID__c: custom1,
+              LastName: "Smith",
+            },
+            oldGroups: [],
+            newGroups: [],
+            toDelete: false,
+            record: null,
+          },
+        ],
+      })
+    ).rejects.toThrow(
+      /Reference data syncing, all related options must be set/
+    );
+  });
+
+  test("cannot sync if only one Group option is set", async () => {
+    const badDestinationOptions = {
+      recordObject: "Contact",
+      recordMatchField: "Custom_External_ID__c",
+      groupObject: "Campaign",
+    };
+
+    await expect(
+      exportBatch({
+        appId,
+        appOptions,
+        destinationOptions: badDestinationOptions,
+        syncOperations,
+        exports: [
+          {
+            recordId: id1,
+            oldRecordProperties: {},
+            newRecordProperties: {
+              Custom_External_ID__c: custom1,
+              LastName: "Smith",
+            },
+            oldGroups: [],
+            newGroups: [],
```

Comment:
let's have a test where we send in some groups (new and old) here and the code shouldn't do anything because the group options are not set. i'm still expecting some changes to `SalesforceExporter` code like `addToGroups` and anywhere there is code like `{ membershipObject, membershipRecordField, membershipGroupField }` or `{ groupObject, groupNameField }`, etc

I think maybe a new test file is a good idea.